versions/kruise-rollout/0.3/values.yaml

```diff
@@ -0,0 +1,57 @@
+# Default values for rollout.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# values for rollout installation
+installation:
+  namespace: kruise-rollout
+  createNamespace: true
+
+rollout:
+  fullname: kruise-rollout-controller-manager
+  healthBindPort: "8081"
+  metricsBindAddr: "127.0.0.1:8080"
+  webhook:
+    port: 9876
+    objectSelector:
+      - key: control-plane
+        operator: NotIn
+        values:
+          - kruise-rollout-controller-manager
+  log:
+    level: 3
+  featureGates: AdvancedDeployment=true
+
+replicaCount: 2
```

Comment:
suggest to set the default replicaCount to be 3 for HA

Reply:
why 3？ I think 2 is enough in most cases.